File: librarymanager-be/src/main/java/ru/elenacherev/librarymanager/api/dto/Reader.java
package ru.elenacherev.librarymanager.api.dto;

import lombok.Builder;
import lombok.Getter;

import java.util.Date;
import java.util.Set;

@Builder
@Getter
public class Reader {
    private Long readrId;
    
    private String firstName; //имя
    private String surname; //фамилия
    private String patronymic; //отчество
    private Date birthdate; //дата рождения, чтобы камасутру мелким не выдать
    private Date regDate;
    private String telephone; //телефон, чтобы присылать смс о готовых заказах и задолжностях
    private String email; //электронная почта. Уникальное поле, используется для авторизации
    
    private Set<Long> bookUsingIds;
    
    private Set<Long> orderIds;
}

File: librarymanager-api/src/main/java/ru/elenacherev/librarymanager/api/dto/Reader.kt
package ru.elenacherev.librarymanager.api.dto

import java.util.*

data class Reader(
    var readrId: UUID? = null,
    val firstName: String,
    val surname: String? = null,
    val patronymic: String? = null,
    val birthdate: Date,
    val regDate: Date,
    val telephone: String?,
    val email: String,
    val bookUsingIds: List<UUID>,
    val orderIds: List<UUID>
)